classes/ActionScheduler_AdminView.php

```diff
@@ -91,6 +91,7 @@ public function process_admin_ui() {
 	 */
 	public function render_admin_ui() {
 		$table = new ActionScheduler_ListTable( ActionScheduler::store(), ActionScheduler::logger(), ActionScheduler::runner() );
+		$table->process_actions();
```

Comment:
Should the call to `process_actions()` be removed from `process_admin_ui()`? It looks like this will fire twice when going to Tools > Scheduled Actions.

Reply:
@jeffstieler Thanks for the suggestion. The PR has been updated.